asv/util.py

```diff
@@ -1127,9 +1147,9 @@ def format_text_table(rows, num_headers=0,
 
 def _datetime_to_timestamp(dt, divisor):
     delta = dt - datetime.datetime(1970, 1, 1)
-    microseconds = (delta.days * 86400 + delta.seconds) * 10**6 + delta.microseconds
+    microseconds = ((delta.days * 86400 + delta.seconds) * 10**6 + delta.microseconds)
```

Comment:
Spaces around the ** operator

Reply:
I don't think spaces are foolish, and flake8 doesn't point to it as a mistake.